packages/components/src/Input/Input.js

```diff
@@ -0,0 +1,92 @@
+import { useState, forwardRef } from 'react';
+import { createStyleSheet } from '@platzily-ui/styling';
+import PropTypes from 'prop-types';
+
+const useStyleSheet = createStyleSheet(
+  (theme, { required, type }) => ({
+    inputDefault: {
+      width: 199,
+      height: 40,
+      borderRadius: 5,
+      display: 'flex',
+      alignContent: 'center',
+      alignItems: 'center',
+      justifyContent: 'center',
+      backgroundColor: theme.palette.neutral.light,
+      borderWidth: 1,
+      borderStyle: 'solid',
+      borderColor: theme.palette.neutral.secondary,
+      required,
+      type,
+      '&:focus': {
+        borderColor: theme.palette.tertiary.main,
+        backgroundColor: theme.palette.neutral.light,
+        borderWidth: 1,
+        borderStyle: 'solid',
+        outline: 'none',
+      },
+      '&:disabled': {
+        borderColor: theme.palette.neutral.tertiary,
+        cursor: 'not-allowed',
+      },
+      '&:invalid': {
+        borderColor: theme.palette.error.main,
+      },
+      '&:hover': {
+        borderColor: theme.palette.tertiary.main,
+      },
+    },
+    inputRequired: {
+      textAlign: 'left',
+      '&::placeholder': {
+        color: theme.palette.error.main,
+        paddingLeft: '95%'
+      }
+    },
+  }),
+  { key: 'InputComponent' },
+);
+
+const Input = forwardRef(function Input(props, ref) {
+  const { required, className, ...otherProps } = props;
+  const { classes, cx } = useStyleSheet(props);
+
+  // useState Hook
+  const [state, setState] = useState({
+    name: '',
+  });
+  const handleInputChange = (event) => setState({ name: event.target.value });
+
+  // Required Validation because of the styles
+  let requiredStyles;
+  let placeHolder;
+  if(required) {
+    requiredStyles = classes.inputRequired;
+    placeHolder = '*';
```

Comment:
I think we should remove this, lets let the users handle that asterisk theirselves.